content/actions/guides/about-continuous-integration.md

```diff
@@ -82,13 +82,13 @@ You won't be able to merge the pull request if your repository is configured to
 
 ## Status badges for workflow runs
 
-{% data reusables.repositories.actions-workflow-status-badge-into %}
+{% data reusables.repositories.actions-workflow-status-badge-intro %}
 
-For more information, see "[Configuring a workflow](/articles/configuring-a-workflow)."
+For more information, see "[Adding a workflow status badge](/actions/managing-workflow-runs/adding-a-workflow-status-badge)."
 
 ## Further reading
 
-- "[Setting up continuous integration using {% data variables.product.prodname_actions %}](/articles/setting-up-continuous-integration-using-github-actions)"
+- "[Setting up continuous integration using workflow templates](/actions/guides/setting-up-continuous-integration-using-workflow-templates)"
```

Comment:
We actually don't include articles in Further reading that are included in the article's sidebar, as this one is, so we can remove this entirely ([ref](https://github.com/github/docs/blob/main/contributing/content-model.md#further-reading)).

```suggestion
```